frontend/src/components/error/Error.tsx

```diff
@@ -0,0 +1,18 @@
+import Navbar from '../navigation/Navbar'
+import styles from './Error.module.css'
+import ErrorDisplay from './ErrorDisplay'
+
+const ErrorPage = () => {
+    const title = '404'
+    const text = "This page doesn't exists."
```

Comment:
Should we use function parameters ?
```suggestion
const ErrorPage = (title, text) => {
```

Reply:
No.
ErrorPage is set as ErrorElement in createBrowserRouter(). As such every Error thrown in any route will bubble up to this page and we catch them there with a react-router-dom method.